### DISCUSSION TITLE
Student pack (university not existed)

### DISCUSSION THREAD

**Discussion by wework13** [community/community#65785] [GitHub Education] [answered]
### Select Topic Area

Product Feedback

### Body

My university is not in the list of universities. There is no way to get the Github package?
If the university is not in the list, can't we do anything?

**Comment by amirroox** [comment, marked as answer]
Considering that the list is not in the universities, you cannot do anything.
Send a message to support and mention your problem.
You can also choose one of the available universities (no need for email) and send your university documents and mention this in support at the end.

**https://support.github.com/features/education**

**Reply by wework13** [reply]
There is no section to add a university?

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
